wordpress-mobile/WordPress-Android
# Update menu and tab bar icons

This PR updates restructures Menu on MySite, updates icons on menu and icons on Jetpack tab bar.
See reference https://a8c.slack.com/archives/C04SFL0RP51

|Before | After | After|
|-|-|-|
|![Screenshot_20230308_135647](https://user-images.githubusercontent.com/990349/223618862-2178f594-24e4-4113-a4db-bceacec19b8a.png)|![Screenshot_20230309_141459](https://user-images.githubusercontent.com/990349/223907371-2eb9f198-3e68-45e5-b35d-8280ab764e19.png)|![Screenshot_20230309_141510](https://user-images.githubusercontent.com/990349/223907411-0afb5771-3c11-414f-8573-b956c329c323.png)|



Fixes #


To test:

Test 1
- Launch WP app
- Go to Menu tab if not already selected
- Verify that the Menu is now organized into Content, Traffic, and Manage categories
- Verify that the new icons are showing up in place of old ones as shown above

Test 2
- Launch Jetpack app
- Go to Menu tab if not already selected
- Verify that the Menu is now organized into Content, Traffic, and Manage categories
- Verify that the tab bar icons match the ones shown in after images above.  For e.g. My Site has Jetpack icon

>**Note**
> Some menu items are removed for e.g. View Site
> Some are renamed for e.g. View Admin is now WP Admin
> Related menu items under manage have been grouped with spacing in between since there are a large number of items

## Regression Notes
1. Potential unintended areas of impact
Look for anything missing

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual and unit tests

3. What automated tests I added (or what prevented me from doing so)
Updated unit tests

PR submission checklist:
